ws/mocks/messages.go

```diff
@@ -0,0 +1,27 @@
+package mocks
+
+import (
+	"errors"
+
+	"github.com/mainflux/mainflux"
+)
+
+var _ mainflux.MessagePublisher = (*publisher)(nil)
+
+var (
```

Comment:
Either group it with the type check above, or drop the group completely. No need for having a one-member variable group.